### PR TITLE
Make fields as required

### LINKED ISSUE #14: Visitor form validation
- [x]  Make field as required.
- [x]  Country list as a dropdown.

### PR DESCRIPTION
**Pull request by sangeethailango**
Fixed #14

### CODE REVIEW

File: priv/repo/migrations/20240226105538_add_field_to_visitor.exs
Comment (sakthibalan15) [inline priv/repo/migrations/20240226105538_add_field_to_visitor.exs:1]: Why this migration file is needed?

Comment (sangeethailango) [reply priv/repo/migrations/20240226105538_add_field_to_visitor.exs:1]: Will delete it.

Comment (sangeethailango) [reply priv/repo/migrations/20240226105538_add_field_to_visitor.exs:1]: @sakthibalan15 deleted it.

File: lib/mmbooking_Clone_web/live/visitor_live/new_visitor.ex [hunk 1]
```diff
@@ -4,6 +4,244 @@ defmodule Mmbooking_CloneWeb.VisitorLive.NewVisitor do
   alias Mmbooking_Clone.User
   alias Mmbooking_CloneWeb.VisitorLive.AcknowledgementEmail
   def mount(_params, _session, socket) do
+    countries = ["Afghanistan",
```
Comment (sakthibalan15) [inline lib/mmbooking_Clone_web/live/visitor_live/new_visitor.ex:7]: call this in a separate function.

Comment (sangeethailango) [reply lib/mmbooking_Clone_web/live/visitor_live/new_visitor.ex:7]: ok

**Pull request merged**